### ISSUE TITLE
layout_fruchterman_reingold 'repulserad' argument 0.8.0 vs 0.7.1.post6 disappeared

### ISSUE THREAD

**Issue by Bugro** [igraph/python-igraph#282]
Hello,

I was using Fruchterman Reingold on an existing graph as follows with python-igraph version 0.7.1.post6: 
```python
graph.layout_fruchterman_reingold(maxiter=1000, repulserad=N).
```

After upgrading to 0.8.0 I got the following error:
```python
TypeError: 'maxiter' is an invalid keyword argument for this function
```

I understood from the R documentation [here](https://igraph.org/r/doc/layout_with_fr.html) that maxiter was renamed to niter.

But then I got a new error with 'repulserad'.
```python
TypeError: 'repulserad' is an invalid keyword argument for this function
```

Again, R documentation (I could not find this information in python doc) specifies that this argument is not supported in 0.8.0 (along with other arguments) but then **how can I obtain the behavior I used to have in 0.7.1.post6? Is it possible to obtain a repulsion force in 0.8.0? Should I better switch back to the former release?**

Thank you in advance,
Guillaume

**Comment by ntamas** [comment]
Indeed, `maxiter` was renamed to `niter` in the C core and the Python interface followed this change. In the future we will try to be more cautious about changes like this and provide deprecation notices.

As for `repulserad`, this was removed completely from the C core so I had to remove it from the Python interface as well (the Python interface simply calls the appropriate function in the C library so there's not much I can do on the Python side). The repulsive force is still present in the algorithm, but it is not tunable any more to bring the implementation closer to the "real" Fruchterman-Reingold layout that was published in the original paper.

Anyway, I'm afraid that the only way to get the argument back is to revert to 0.7.1post6.
